### DISCUSSION TITLE
v3.0.2 (Beta)

### DISCUSSION THREAD

**Discussion by osy** [utmapp/UTM#3464] [Releases]
## Installation
Visit [https://getutm.app/install/][1] for the most up to date installation instructions.

## Highlights
* **Wizard for VM creation**: Easily create a new VM with the correct default settings with a guided step-by-step wizard. Preview: https://twitter.com/UTMapp/status/1475606159428046854
* **QEMU v6.2.0**: Updated QEMU backend to the latest release. Full changelog: https://wiki.qemu.org/ChangeLog/6.2
* (iOS 13+) **New toolbar**: The new toolbar is much more compact and friendly for smaller displays. It is also now movable and does not interfere with the iOS 15 multitasking button on the top center of the screen. Preview: https://twitter.com/UTMapp/status/1407350167494443023
* (macOS 12+, ARM64) **Virtualize macOS 12**: New integration with Apple's Virtualization.framework backend allows native hardware accelerated virtualization of macOS 12 (supporting both CPU and GPU virtualization). UTM can also download the latest macOS IPSW directly from Apple. (This is only supported on macOS 12+ running on an ARM64 Mac.)
* (macOS 11+) **Virtualize Linux with Apple backend**: You now have the option to use Apple's Virtualization.framework to virtualize Linux. Note that while there are some performance benefits, it is still recommended to use the QEMU backend as it is more stable, has more device support (including graphical display), and is easier to configure.

## Notes
* The default CPU for ARM64 virtual machine has changed from `cortex-a72` to `default`. This is the recommended option and you should change your existing VMs to use it. When using virtualization on M1(X) macOS, the `cortex-a72` CPU is known to have compatibility issues which causes the VM to sometimes freeze (including during Windows install). `default` will use the `host` CPU model when virtualization is used and `cortex-a72` when emulation is used.
* The GL hardware acceleration support is now marked experimental to highlight that it is not stable. It is also no longer the default option. If you are experiencing lots of crashes, make sure that you chose a display hardware that is **not** marked as "GPU Supported".
* The cursor and resolution change issues on ARM64 Windows has been fixed in [SPICE tools 0.164.3][3].

## Changes (v3.0.2)
* (macOS) Fixed drive delete/move buttons not appearing on macOS 11 (#3461)
* (macOS) Added missing network entitlement for bridged networking on Apple VMs
* (macOS) Wizard: convert VHDX to QCOW2 when creating a Windows VM
* (macOS) Any imported drive on an interface that is NOT floppy, pflash, or none will be converted to QCOW2. This does not apply to removable drives.
* (iOS) Improved toolbar visibility and dimming while idle behaviour
* Wizard: skip drive creation if rootfs image is selected
* Fixed an issue where an imported drive does not use the selected interface

## Changes (v3.0.1)
* (macOS) Fixed launching on macOS 11 (#3433)
* (macOS) Fixed "auxiliaryStorage is nil" when creating macOS VM on M1 Mac (thanks @pointum) (#3429)
* (iOS) Proper enablement of extended RAM usage on M1 iPad 16GB (thanks @kkk669)
* (iOS) Fixed crash on VM launch on < iOS 13 due to toolbar not being initialized
* (iOS) Version information will now be shown in the system Settings app for UTM (at the bottom)
* Fixed cursor and screen resolution issues on ARM64 Windows after installing SPICE tools. Requires updating SPICE tools to [latest version][3].
* Add virtio-rng-pci device to fix boot warning about randomness source (thanks @ktprograms)
* Default upscaling mode is now "nearest neighbor"
* Fixed floppy drives being read-only (thanks @conath) (#3333)
* Wizard: no longer use NVMe for Windows on x86_64 systems
* Fixed a regression where Windows boots into BSOD often on x86_64 virtualization
* Updated localization for Zh-Hant (thanks @pan93412)
* Updated localization for Zh-Hans (thanks @js-john)
* Fixed an issue where sometimes a VM will not boot due to invalid drive id (thanks @conath) (#3388)
* Changing target to PC (i440FX) will now turn off hypervisor on x86_64 by default because it is unstable (#2420)

## Changes (v3.0.0)
* Added VM creation wizard
* Updated QEMU to v6.2.0
* Updated ANGLE to latest commit
* Updated virglrenderer to latest commit (this may fix #2743)
* Added `default` and `host` CPU option for ARM64 virtual machines. This will be the default for newly created VMs.
* The downloader will no longer perform CRC checks on the ZIP file and this will significantly speed up extraction.
* (iOS 13+) Redesigned toolbar (#496)
* (macOS) Redesigned VM settings
* (macOS) Fix crash moving drives (#3117)
* (macOS) Make HVF per-VM configurable (#2493)
* (macOS) Add Apple Virtualization.framework backend
* (macOS) New Terminal frontend with SwiftTerm, currently only used with Apple backend on macOS
* (macOS) Apple Virtualization supports virtual serial port through PTY device

## Issues
Please check the full list on [Github][2] and help report any bug you find that is not listed.

[1]: https://getutm.app/install/
[2]: https://github.com/utmapp/UTM/issues
[3]: https://mac.getutm.app/support/

<hr /><em>This discussion was created from the release <a href='https://github.com/utmapp/UTM/releases/tag/v3.0.2'>v3.0.2 (Beta)</a>.</em>

**Comment by Fede90mdq** [comment]
Hi! I noticed that it fails when I attempt to install external app via iso image. If i mount an iso image to install an app, the iso fails at the middle of the installation and disconnect the image mounted...so it fails the installation. It happens with Win11(arm) ALWAYS and, with Win10(arm), it occurs sometimes (sometimes i can do the complete installation with no problem).
This problem appears with UTM 3.0.0, 3.0.1 & 3.0.2. Previous versions doesn't have that bug.

Thanks a lot community, UTM is excellent for me to virtualize and lastest versions work really fast!

**Reply by osy** [reply]
Does v2.4.1 have the problem? It may be related to #3216

**Reply by conath** [reply]
Agree this sounds like a known issue.

**Reply by Fede90mdq** [reply]
Great! With XHCI worked perfectly 👏🏼

**Reply by osy** [reply]
Sorry are you referring to v2.4.1?

**Reply by Fede90mdq** [reply]
I didn't have that problem in 2.4.1...today i activated XHCI in this beta 3.0.2 and worked perfectly.

**Reply by Fede90mdq** [reply]
Buddies, I have this problem again with the new 3.0.3 version (using XHCI).

**Reply by conath** [reply]
Unfortunately the problem has not been resolved in any UTM version @Fede90mdq. The Windows install works sometimes and fails sometimes (more often). We don’t have a workaround available other than: you need to try again until it works.

**Reply by Fede90mdq** [reply]
Thanks for the quick response! If it helps, i noticed that Win10 fails less than Win11 (ARM both).
